doozer/doozerlib/backend/konflux_image_builder.py

```
@@ -249,16 +249,25 @@ async def build(self, metadata: ImageMetadata, git_auth_secret: Optional[str] =
-                    except Exception as e:
-                        logger.error(
-                            f"Failed to get SLA attestation / source signature from konflux for image {definitive_image_pullspec}, marking build as {KonfluxBuildOutcome.FAILURE}. Error: {e}"
+                    # Skip for non-OCP groups (e.g., OKD) as they may not have attestations/signatures
+                    is_ocp_group = self._config.group_name.startswith("openshift-")
+                    if is_ocp_group:
+                        try:
+                            # use image_digest here to be precise, image_pullspec can collide in case of golang-builder images
+                            await self._validate_build_attestation_and_signature(
+                                definitive_image_pullspec, metadata.distgit_key
+                            )
+                        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)